tests/Aspire.Pomelo.EntityFrameworkCore.MySql.Tests/ConformanceTests.cs

```diff
@@ -119,28 +123,14 @@ public void DbContextCanBeAlwaysResolved()
         Assert.NotNull(dbContext);
     }
 
-    [ConditionalFact]
-    public void TracingEnablesTheRightActivitySource()
-    {
-        SkipIfCanNotConnectToServer();
-
-        RemoteExecutor.Invoke(() => ActivitySourceTest(key: null)).Dispose();
-    }
+    [RequiresDockerFact]
+    public Task TracingEnablesTheRightActivitySource()
+        => RunWithFixtureAsync(obj => obj.ActivitySourceTest(key: null));
```

Comment:
```suggestion
        => RemoteExecutor.Invoke(() => RunWithFixtureAsync(obj => obj.ActivitySourceTest(key: null))).Dispose();
```

Reply:
Has this been addressed?

Reply:
Sorry! I kept looking at the other instance of the same thing. Fixing now.